app/assets/javascripts/miq_policy/import.js

```diff
@@ -1,4 +1,4 @@
-//= require_directory ../SlickGrid-2.1/
+//= require_directory slickgrid
```

Comment:
Same here: Is this needed and is this right since the directory now comes from a gem?